eq-author/src/apollo/sentryUtils.test.js

```diff
@@ -0,0 +1,39 @@
+import { setSentryUser, setSentryTag, sendSentryError } from "./sentryUtils";
+import * as Sentry from "@sentry/browser";
+import uuid from "uuid";
+import jwt from "jsonwebtoken";
+
+jest.mock("@sentry/browser");
+
+const createAccessToken = (token, signingKey = uuid.v4()) => {
+  return jwt.sign(token, signingKey);
+};
+
+describe("sentry utils functions", () => {
+  let networkError = {
+    fail: "Uh oh",
+  };
+
+  it("should call sentry with email and username", () => {
```

Comment:
Can we add test when the token is missing or invalid

Reply:
I think this will be covered by the test checking if the user exists in the system but but Ive added a test to make sure it does the same thing as that when the token is missing